content/problem-frame-and-identity-focus/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Problem Framing and Identity Focus
+content_type: project
+submission_type: link 
+ready: true

```

Comment:
ready:true should always be removed.

Please remember to run `npm run validate` in the terminal